### ISSUE TITLE
Cannot update from 0.7.9 to 0.9.2

### ISSUE THREAD

**Issue by MrJozroz** [dsafa/audio-band#197]
I tried installing the new version but the old version keeps popping up instead. Not really sure what more to say about the issue.

**Comment by dsafa** [comment]
Yeah, the new installer doesn't detect the old installer script but it should overwrite it, so if you restart explorer windows should load the new version. If that doesn't work then uninstalling the old version first should get rid of it.

I'll update the text to clarify.
